tf-psa-crypto/.gitignore

```diff
@@ -1 +1,12 @@
 Makefile
+
+programs/psa/aead_demo
```

Comment:
Why in `tf-psa-crypto/.gitignore` and not in `tf-psa-crypto/programs/.gitignore`?

Reply:
Sorry, resolved now.